### PR TITLE
Day_2 -

### PR DESCRIPTION
**Pull request by sheylaPozo**
> In the following pull request
> 
> -We have implemented a class to add, remove and display our information in the awesome library.
> -We have added a method for adding books to our library
> -We have added a method for removing books from our list of books
> -We have added an event Listener for the form for local storage and for the remove button on the actual list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ GitHub: [@Munsa1](https://github.com/munsa1)
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Munsa1/Awesome-Books/issues)
 
 ## Show your support
```
Comment (carlos-ssh) [inline README.md:70]: - Add a descriptive README file. Provide information about your project.
- Kindly fix linters issues in your branch.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @sheylaPozo  & @Munsa1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Munsa1) [reply README.md:70]: thank you for the feedback

**Review by carlos-ssh — APPROVED** [review]
Hi @sheylaPozo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

[OPTIONAL]
- Kindly provide descriptive README files. you will find templates [here](https://github.com/microverseinc/readme-template).

- **This is one of your general requirements for your project.**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**